docs/sdk_tutorial.md

```diff
@@ -13,7 +13,7 @@ session = client.sessions.start(timeout_minutes=5)
 # get the session status
 status = client.sessions.status(session.session_id)
 # list your active sessions
-active_sessions = client.sessions.list()
+active_sessions = client.sessions.list_sessions()
```

Comment:
can we keep `.list()` ? I prefer the shorter syntax + it's compatible with steel & browserbase